FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
+### What state is Quaver currently in?
+
+Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.
```

Comment:
Reword this to: `Quaver is still in early development and is currently in closed alpha. `

Reply:
Done